src/test/java/nextstep/subway/unit/LineNewTest.java

```diff
@@ -0,0 +1,187 @@
+package nextstep.subway.unit;
+
+import nextstep.subway.domain.Line;
+import nextstep.subway.domain.Section;
+import nextstep.subway.domain.Station;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class LineNewTest {
+    @Test
+    void addSection() {
+        Station 강남역 = new Station("강남역");
+        Station 역삼역 = new Station("역삼역");
+        Station 삼성역 = new Station("삼성역");
+        Line line = new Line("2호선", "green");
+
+        line.addSection(강남역, 역삼역, 10, 5);
+        line.addSection(역삼역, 삼성역, 5, 3);
+
+        assertThat(line.getStations()).containsExactly(강남역, 역삼역, 삼성역);
+    }
+
+    @DisplayName("상행 기준으로 목록 중간에 추가할 경우")
+    @Test
+    void addSectionInMiddle() {
+        Station 강남역 = new Station("강남역");
+        Station 역삼역 = new Station("역삼역");
+        Station 삼성역 = new Station("삼성역");
+        Line line = new Line("2호선", "green");
+
+        line.addSection(강남역, 역삼역, 10, 5);
+        line.addSection(강남역, 삼성역, 5, 3);
+
+        assertThat(line.getSections().size()).isEqualTo(2);
+        Section section = line.getSections().stream()
+                .filter(it -> it.getUpStation() == 강남역)
+                .findFirst().orElseThrow(RuntimeException::new);
+        assertThat(section.getDownStation()).isEqualTo(삼성역);
+        assertThat(section.getDistance()).isEqualTo(5);
+        assertThat(section.getDuration()).isEqualTo(3);
+    }
+
+    @DisplayName("하행 기준으로 목록 중간에 추가할 경우")
+    @Test
+    void addSectionInMiddle2() {
+        Station 강남역 = new Station("강남역");
+        Station 역삼역 = new Station("역삼역");
+        Station 삼성역 = new Station("삼성역");
+        Line line = new Line("2호선", "green");
```

Comment:
```suggestion
        Line 2호선 = new Line("2호선", "green");
```

line이라는 이름보다는 의미를 가지는 이름으로 변경해 보면 어떨까요? 😄